q2_feature_table/plugin_setup.py

```diff
@@ -18,7 +18,14 @@
     name='feature-table',
     version=q2_feature_table.__version__,
     website='https://github.com/qiime2/q2-feature-table',
-    package='q2_feature_table'
+    package='q2_feature_table',
+    citation_text=('The Biological Observation Matrix (BIOM) format or: how '
```

Comment:
I'm ok with it here as a lot of the operations in this plugin use the BIOM API. When we transition to having method/visualizer-specific plugins, maybe at that time we could move it from being a plugin citation to a citation for the relevant methods/visualizers.